resources/html/graddPayloadForm.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <head>
+    </head>
+    <body>
+        <form method="post" action="/missions/updateGraddPayload">
```

Comment:
Are you sure this URL works? Looks like it's pointing to the same page. Shouldn't this point to the `missioncontrol` API URL instead?

Reply:
you're right
how about giving me the URLs for missioncontrol and missions?
I need to understand the URL structure - and I wasn't explained it

Reply:
To get the missioncontrol base URL - please see how that's done in `/missions/src/lib/api.js`

Reply:
ok I got it
fixing now